### ISSUE TITLE
Cadence/Temporal unit tests failing: cannot connect to server

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1200]
Both Cadence and Temporal unit tests are all failing due to being unable to connect to the server.  This is happening on my developer workstation as well as the GitHub test runners.

I've marked these tests with `[TestTrait.Category, TestTrait.Investigate]` to disable them in the GitHub runners.

@johncburns1: Perhaps you could have a look at some point.  I'm on the tail end of the GitHub Action automation effort and just noticed this when retesting the unit testing actions.  The unit tests were basically hanging on these and the GitHub Action eventually gets cancelled.

**Comment by jefflill** [comment]
You know, this is probably the same as #1199.

I'm going to try upgrading to the new .NET 5.0 packages across all of the libraries and modify the test projects to multi-target .NET Framework, .NET Core 3.1 and .NET 5.0 and see that happens.

**Comment by jefflill** [comment]
There's actually a few other issues here including port conflicts with containers hanging around from previous unit tests and docker being put into swarm mode by some unit tests and not restoring normal mode.  I'm not sure why we haven encountered and of this before.

I've added the new `TestHelper.ResetDocker()` method and am going to call this in the constructor of any tests that rely on Docker to ensure it's in a known state.

I also noticed that Visual Studio makes it easy to have unit test projects target multiple frameworks.  I didn't know this was a feature.  I'm going to add net48 and netcoreapp3.1 targets where it makes sense.
